lib/tpm2_policy.h

```diff
@@ -127,4 +127,17 @@ bool tpm2_policy_get_digest(TSS2_SYS_CONTEXT *sapi_context,
  */
 bool tpm2_policy_parse_policy_list(char *str, TPML_DIGEST *policy_list);
 
+/**
+ * Policy to restrict tpm object authorization to specific commands
+ *
+ * @param sapi_context
+ *   The system api context
+ * @param policy_session
+ *   The policy session into which the policy digest is extended into
+ * @param[in] command_code
```

Comment:
in, we don't use this decorator. Normally this is reserved for pointer arguments IIUC as well.